panel/src/components/Blocks/Block.vue

```diff
@@ -220,4 +256,8 @@ export default {
 .k-block-container[data-hidden] .k-block {
   opacity: .25;
 }
+.k-drawer-options .k-button[data-disabled] {
+  vertical-align: middle;
+  display: inline-grid;
+}
```

Comment:
When disabled, the button was converted to span and the appearance was broken.

Reply:
I was tripping over the same problem. Great catch!